style.css

```diff
@@ -1,825 +1,810 @@
-    font-size: 24px;
-    letter-spacing: 2px;
-    margin: 5px 41px;
-  }
+.third__container {
+	width: 256px;
+	height: 115px;
+	margin: 0 auto;
+}
 
-  .header__menu {
-    height: 74px;
-    width: 74px;
-  }
+.third__title {
+  font-size: 20px;
+  line-height: 1;
+  margin-bottom: 0;
+  letter-spacing: .4px;
+  margin-top: 24px;
+}
 
-  .header__menu-item {
-    height: 3px;
-    width: 28.7px;
-    margin-bottom: 4px;
-  }
+.third__subtitle {
+  font-size: 10px;
+  line-height: 1;
+  letter-spacing: 0.07em;
+  opacity: .5;
+  margin-top: 7px;
+  margin-bottom: 11px;
+}
 
-  .header__language {
-    left: 29.5%;
-    top: 34px;
-    font-size: 16px;
-  }
+.third__text {
+	font-size: 9px;
+	line-height: 1.44;
+	opacity: .5;
+}
 
-  .header__vector {
-    left: 32.5%;
-    top: 42px;
-  }
+.third__add {
+	font-weight: 500;
+  font-size: 13px;
+  text-decoration: none;
+  color: #000;
+  padding: 10px 48px;
+  border: 1px solid #000;
+  margin-left: 23px;
+}
 
-  .img__img {
-    width: 439px;
-    height: 450px;
-  }
+.third__next{
+	font-weight: 500;
+  font-size: 15px;
+  line-height: 1.2;
+  position: absolute;
+  width: 147px;
+  top: 507px;
+  right: 40px;
+}
 
-  .step {
-    position: absolute;
-    width: 100%;
-    display: grid;
-    grid-template-columns: 439px 439px;
-    grid-template-rows: 450px;
-    justify-content: center;
-    top: 135px;
-    z-index: 2;
-    grid-column-gap: 42px;
-  }
+@media (min-width: 460px) {
+	.head {
+		min-height: 700px;
+	}
 
-  .text {
-    height: 450px;
+	.head__img-title {
+    font-size: 65px;
+    padding-top: 509px;
   }
+}
 
-  .text__title {
-    font-size: 60px;
-    line-height: 68px;
-    letter-spacing: 4px;
-    width: 439px;
-  }
+@media (min-width: 560px) {
+	.head {
+		min-height: 890px;
+	}
 
-  .text__container {
-    padding: 39px 0;
+	.head__img-title {
+    font-size: 85px;
+    padding-top: 652px;
   }
+}
 
-  .text__subtitle {
-    font-size: 12px;
-    line-height: 16px;
-    letter-spacing: 1.1px;
-    margin-bottom: 40px;
+@media (min-width: 825px) {
+	.head__img-title {
+    font-size: 99px;
+    padding-top: 602px;
   }
+}
 
-  .collection__next-text {
-    font-size: 11px;
-    padding: 10px 27px;
-    margin-right: 43px;
-    letter-spacing: 0;
-  }
+@media (min-width: 957px) {
+	.header__inner-logo-mob,
+	.head__mobile,
+	.second__bag,
+	.third__img {
+    display: none;
+	}
 
-  .collection__other-text {
-    font-size: 11px;
-    letter-spacing: 0;
-    margin-right: 15px;
+	.header {
+	  background-color: none;
+    width: 100%;
   }
 
-  .collection__vector {
-    width: 35px;
+  .header__inner {
+    position: absolute;
+    width: 100%;
+    height: 94px;
   }
 
-  .collection__scroll {
-    width: 22px;
-    height: 39px;
-    left: 46.5%;
-    top: 650px;
+  .header__inner-logo {
+    margin-top: 17px;
+    margin-left: 29px;
   }
 
-  .collection__scroll-elem {
-    left: 47.2%;
-    top: 659.5px;
+  .header__inner-menu-btn {
+	  width: 92px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)